config_src/external/Icepack_interfaces/icepack_tracers.F90

```diff
@@ -0,0 +1,190 @@
+!=======================================================================
+! Indices and flags associated with the tracer infrastructure.
+! Grid-dependent and max_trcr-dependent arrays are declared in ice_state.F90.
+!
+! author Elizabeth C. Hunke, LANL
+
+      module icepack_tracers
+
+      use icepack_kinds
+
+      implicit none
+
+      private
+      public :: icepack_init_tracer_indices
+      public :: icepack_init_tracer_sizes
+      public :: icepack_query_tracer_sizes
+
+      real(kind=dbl_kind), parameter, public :: n_iso=0,n_aero=0
```

Comment:
This line (18) is causing the gnu compiler to fail, because that compiler does not permit reals to be used when declaring the size of memory.  However it does successfully compile if this line is changed to  `integer (kind=int_kind), parameter, public :: n_iso=0, n_aero=0`.